apps/crn-server/test/controllers/research-outputs.test.ts

```diff
@@ -150,9 +155,11 @@ describe('ResearchOutputs controller', () => {
 
       const researchOutputCreateDataObject =
         getResearchOutputCreateDataObject();
-      expect(researchOutputDataProviderMock.create).toBeCalledWith(
-        researchOutputCreateDataObject,
-      );
+      expect(researchOutputDataProviderMock.create).toBeCalledWith({
+        ...researchOutputCreateDataObject,
+        addedDate: mockDate.toISOString(),
```

Comment:
as per the types the addedDate is an optional parameter which CAN be defaulted to current time - I think this test should continue to use the data from the fixture.

We should have another tests that asserts that when the addedDate is missing it defaults it to the current time

Reply:
I thought the controller wouldn't accept the date in its params. It would always set it to current time. Considering that, this test is actually doing the second scenario. It's checking if provider was called with current time. If the controller needs to also accept the parameter from outside then we need to add the first scnearion